src/components/Appointments/Pending.jsx

```diff
@@ -8,6 +8,40 @@ class PendingAppointments extends Component {
     this.state = {
       pending: []
     }
+    this.handleRemoveApt = this.handleRemoveApt.bind(this);
+  }
+
+  handleRemoveApt(event) {
+    const id = event.currentTarget.getAttribute('id');
+    const instructorid = event.currentTarget.getAttribute('instructorid');
+    const time = event.currentTarget.getAttribute('time');
+    const studentName = event.currentTarget.getAttribute('studentName');
+   
+    axios.post('/api/AptRequests/removeApt', {
```

Comment:
Emails should go out after the actual removal of an appointment request. Take this Axios call and put it within the DELETE request